templates/Collector/rules.html.twig

```diff
@@ -0,0 +1,32 @@
+    <span class="sf-toolbar-info-piece-additional sf-toolbar-status">{{ collector.getlogscount }}</span>
+  </a>
+  {% endset %}
+  {% set text %}
+
+  <div class="sf-toolbar-info-piece">
+    <b>Rules logs</b>
+    <span>{{ collector.getlogscount }}</span>
+  </div>
+
+  {% endset %}
+
+  <div class="sf-toolbar-block">
+    <div class="sf-toolbar-icon">{{ icon|default('') }}</div>
+    <div class="sf-toolbar-info">{{ text|default('') }}</div>
+  </div>
+{% endblock %}
+
+{% block panel %}
+  <div class="title">
+    <h2>{{ 'Rules'|t }}</h2>
```

Comment:
Same here and in the link at the beginning of this file. Do we want to run Rules through |t?